ocrolib/common.py

```diff
@@ -665,34 +666,50 @@ def expand_args(args):
     else:
         return args
 
-data_paths = [
+
+_data_paths = [
     ".",
-    "./models",
-    "./data",
-    "./gui",
-    "/usr/local/share/ocropus/models",
-    "/usr/local/share/ocropus/data",
-    "/usr/local/share/ocropus/gui",
-    "/usr/local/share/ocropus",
+    "models",
+    "data",
+    "gui",
 ]
 
-def ocropus_find_file(fname,gz=1):
+
+def ocropus_find_file(fname, gz=True):
     """Search for OCRopus-related files in common OCRopus install
     directories (as well as the current directory)."""
     if os.path.exists(fname):
         return fname
     if gz:
-        if os.path.exists(fname+".gz"):
-            return fname+".gz"
-    for path in data_paths:
-        full = path+"/"+fname
-        if os.path.exists(full): return full
+        if os.path.exists(fname + ".gz"):
+            return fname + ".gz"
+
+    for path in _data_paths:
+        full = os.path.join(path, fname)
+        if os.path.exists(full):
+            return full
+    if gz:
+        for path in _data_paths:
+            full = os.path.join(path, fname + ".gz")
+            if os.path.exists(full):
+                return full
+
+    share_path = os.path.join(sysconfig.get_config_var("datarootdir"),
```

Comment:
Is the [`finddir`](https://github.com/tmbdev/ocropy/blob/master/ocrolib/common.py#L607) method used anywhere? Should the data-loading logic be moved to [`defaults.py`](https://github.com/tmbdev/ocropy/blob/master/ocrolib/default.py)?